Jenkinsfile

```diff
@@ -53,6 +53,11 @@ pipeline {
         sh 'rm -rf build'
         sh 'rm Example/Release-keys.xcconfig'
       }
+      post {
+        always {
+          sh 'rm -rf Example/Keys.plist || true'
```

Comment:
You don't need the `-r`, maybe not even the `-f`. It's not a directory so it's not recursive and I doubt you need to force it.